docs/painless/painless-contexts/painless-watcher-context-example.asciidoc

```diff
@@ -0,0 +1,112 @@
+==== Complete Example
+
+While these examples are contrived, a more complete watch shown below contains
```

Comment:
Don't think you need to call out the example as contrived as that makes it sound like it's not really useful.  Short examples often don't represent real world scenarios.